lib/rootview.dart

```diff
@@ -50,6 +56,8 @@ class RootView extends StatelessWidget {
         items: tabBarItems
     );
 
+    _ensureLoggedIn();
```

Comment:
is this method blocking? I know it _feels_ blocking but I think it just returns a `Future`